app/views/submissions/steps/population/_step01.html.haml

```diff
@@ -25,5 +25,16 @@
   %small.help-block Please, choose the type of population you are submitting.
 
 .form-group
+  - organisations = PlantPopulation.visible(current_user.id).attribute_values(:establishing_organisation).delete_if{|o| o.blank?}.sort
```

Comment:
Please move it to the Step01ContentForm class and leave only the options_for_select call in the view code.

Also, try to use pluck to get only a single column value from the DB, then I think you'll need to call uniq to remove duplicates. You can also use the ampersand collection operator to call blank in the block.